app/views/billing_projects/access_request.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1 class="text-center">Additional Access Required</h1>
+<div id='billing-access-request'>
+  <p><big>
+    In order to manage your Terra billing projects, Single Cell Portal must ask for read access to the list of your Google
+    Cloud Platform billing accounts.  This is controlled via the Google sign-in and consent process, so you must re-authenticate
+    and grant access to Single Cell Portal to be able to read this data.</big>
+  </p>
+  <p><big>
+    <strong>Please note:</strong> <u>Single Cell Portal will not be granted access to make billing requests on your behalf,
+    or to manage your billing accounts in any way.</u>  This is simply a request to see a list of billing account names.
+  </big>
+  </p>
+  <p><big>
+    If you wish to continue, please click the 'Sign In and Grant Access' button below and complete the sign-in process,
+    or you may decline and return to the home page.</big>
```

Comment:
At demo, Tim or Amelia suggested less depth in these messages, which I agree would help. Perhaps:

```suggestion
    To manage your Terra billing projects, Single Cell Portal needs to know your Google
    Cloud Platform billing account names.  This is mediated by Google, so please re-authenticate
    and grant that access to Single Cell Portal.</big>
  </p>
  <p><big>
    <u>Single Cell Portal will not be granted access to make billing requests for you,
    nor to manage your billing accounts.</u>  This is simply a request to see billing account names.
  </big>
  </p>
  <p><big>
    To continue, please click the "Sign In and Grant Access" button below and complete signing in,
    or you may decline and return to the home page.</big>
```